test/integration/policy-imagemanifest_test.go

```diff
@@ -0,0 +1,158 @@
+			common.GvrPolicy,
+			userNamespace+"."+policyIMVName,
+			clusterNamespace,
+			true,
+			defaultTimeoutSeconds,
+		)
+		Expect(managedPolicy).NotTo(BeNil())
+	})
+
+	It("stable/"+policyIMVName+" should be NonCompliant", func() {
+		By("Checking if the status of the root policy is NonCompliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyIMVName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("Enforcing stable/"+policyIMVName, func() {
+		By("Patching remediationAction = enforce on the root policy")
+		_, err := clientHubDynamic.Resource(common.GvrPolicy).Namespace(userNamespace).Patch(
+			context.TODO(),
+			policyIMVName,
+			k8stypes.JSONPatchType,
+			[]byte(`[{"op": "replace", "path": "/spec/remediationAction", "value": "enforce"}]`),
+			metav1.PatchOptions{},
+		)
+		Expect(err).To(BeNil())
+	})
+
+	It("The subscription should exist", func() {
+		By("Checking the subscription in the " + operatorNS + " namespace")
+		Eventually(
+			func() string {
+				output, _ := common.OcHub("get", "subscriptions.operators.coreos.com", "-n",
+					operatorNS, subName, "-o", "name")
+
+				return output
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal("subscription.operators.coreos.com/container-security-operator"))
+	})
+
+	It("The operator should exist", func() {
+		By("Checking the operator in the " + operatorNS + " namespace")
+		Eventually(
+			func() string {
+				output, _ := common.OcHub("get", "operators", subName+"."+operatorNS, "-o", "name")
+
+				return output
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal("operator.operators.coreos.com/container-security-operator.openshift-operators"))
+	})
+
+	It("stable/"+policyIMVName+" should detect imageManifestVulns", func() {
+		By("Checking if the status of the IMV config policy is NonCompliant")
+		Eventually(
+			utils.GetWithTimeout(
+				clientManagedDynamic,
+				common.GvrConfigurationPolicy,
+				"policy-imagemanifestvuln-example-imv",
+				clusterNamespace,
+				true,
+				defaultTimeoutSeconds,
+			).Object["status"].(map[string]interface{})["compliant"],
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("stable/"+policyIMVName+" should be NonCompliant", func() {
+		By("Checking if the status of the root policy is NonCompliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyIMVName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("Cleans up", func() {
+		_, err := utils.KubectlWithOutput(
+			"delete", "-f", policyIMVURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		_, err = utils.KubectlWithOutput(
+			"delete", "subscription", subName, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		csvName, err := utils.KubectlWithOutput(
```

Comment:
This would get all CSVs and delete all of them. You should just get the CSV for imagemanifestvul. You can use the filter. See an example [here](https://github.com/stolostron/governance-policy-framework/blob/ee05e1d4d8b3671c42a5b74dd84122d57fd19984/build/clean-up-cluster.sh#L50)